apps/files_external/src/views/CredentialsDialog.vue

```diff
@@ -0,0 +1,86 @@
+<!--
+  - SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+	<NcDialog :buttons="dialogButtons"
+		class="external-storage-auth"
+		close-on-click-outside
+		data-cy-external-storage-auth
+		is-form
+		:name="t('files_external', 'Enter storage credentials')"
```

Comment:
You can ignore if you like but I think this would be more condensed, but fine with both :)

```suggestion
		:name="t('files_external', 'Storage credentials')"
```